prompb/remote.proto

```diff
@@ -43,3 +65,12 @@ message QueryResult {
   // Samples within a time series must be ordered by time.
   repeated prometheus.TimeSeries timeseries = 1;
 }
+
+// ChunkedReadResponse is a response when response_type equals STREAMED_XOR_CHUNKS.
+// We strictly stream full series by series. Single frame can contain partion of the single series.
```

Comment:
Not sure what "We strictly stream full series by series" means. I thought it means that each single series wouldn't be broken up, but then the next sentence says the opposite?

Reply:
It rather means that we expect series to be be sequentially send. 

So you cannot send series ABC [1 10] then XYZ [1-20] then ABC [11-20] but you can ABC [1 10] then ABC [11-20]  then  XYZ [1-20]  (:

Reply:
Tried to be more explicit. PTAL. Also not 100% sure if we want to be that strict on interface side.